### PR TITLE
Make timer appear again in YT 2018.2

### LINKED ISSUE #1: YouTrack 7
I think the button isn't appearing in the agile popup for YouTrack 7

### PR DESCRIPTION
**Pull request by danmichaelson**
In the agile popup, and in the "experimental" full-screen view.

Closes #1

### CODE REVIEW

File: chrome-extension/youtrack.js [hunk 1]
```diff
@@ -9,12 +9,12 @@
         this.addTimers = bind(this.addTimers, this);
         this.projectNameSelector = [
           ".fsi-property .attribute.bold",            // full-screen view and edit
-          ".sb-settings-criteria, .sb-board-name"     // agile popup (try find project name, if failed - use board name)
+          "yt-issue-project"                          // agile popup, full-screen experimental view and edit
```
Comment (extempl) [inline chrome-extension/youtrack.js:12]: @danmichaelson is it a class? Id?

File: chrome-extension/youtrack.js [hunk 2]
```diff
@@ -125,8 +138,9 @@
         timer.setAttribute("data-project", JSON.stringify(data.project));
         timer.setAttribute("data-item", JSON.stringify(data.item));
         var goAfter =
-                item.querySelector('.star_fsi') ||    // full-screen view and edit ('.issueId' also works pretty well for slightly different position)
-                item.children[0];                     // agile popup
+          item.querySelector('.star_fsi') ||            // full-screen view and edit
+                                                        //   ('.issueId' also works pretty well for slightly different position)
+          item.querySelector('yt-issues-more-actions'); // agile popup
```
Comment (extempl) [inline chrome-extension/youtrack.js:143]: @danmichaelson same here

**Review by extempl — CHANGES_REQUESTED** [review]

File: chrome-extension/youtrack.js (hunk 1 above)
Comment (danmichaelson) [reply chrome-extension/youtrack.js:12]: It's a DOM element type like H1. A custom Angular element in this case. Seems to work 😄

Comment (extempl) [reply chrome-extension/youtrack.js:12]: Oh. I see. Ok then

**Pull request merged**